docs/benchmarks/webrtc.md

```diff
@@ -0,0 +1,32 @@
+# WebRTC
+
+The WebRTC tests conducted for the Jellyfish media server aimed to assess its performance and reliability in handling real-time communication scenarios, replicating real-life online meetings. 
+
+## Setup
+
+The benchmarks use an 8-core Intel Xeon Processor (Skylake, IBRS).
```

Comment:
Let's make this similar to HLS benchmarks i.e. pass info about the other machine and their network conneciton link

Reply:
I don't think the other machine is important for the readers, so I think a short mention of the other machine will suffice.